library/tinkerpop/README.md

```diff
@@ -1,3 +1,22 @@
+# GafferPop
+
+GafferPop is a lightweight Gaffer implementation of the
+[TinkerPop framework](https://tinkerpop.apache.org/), where TinkerPop methods
+are delegated to Gaffer graph operations.
+
+It is still in development. The implementation is basic and its performance is
+unknown in comparison to using standard Gaffer `OperationChains`.
+
+Please see the [official Gaffer documentation](https://gchq.github.io/gaffer-doc/latest/)
+for more information on the current GafferPop implementation, some useful links are as follows:
+
+- [Set up a Gremlin console connection using GafferPop](https://gchq.github.io/gaffer-doc/latest/).
+- [Basic Gremlin querying guide](https://gchq.github.io/gaffer-doc/latest/).
+- [Current limitations of the GafferPop implementation](https://gchq.github.io/gaffer-doc/latest/).
+
+## Copyright
+
+```text

```

Comment:
This section isn't required as we've already got this on the main page and in all the relevant files. I think it's only here because it was in the original README. The other READMEs where content was moved to docs have this removed.